app/src/main/java/com/vitorpamplona/amethyst/ui/screen/ChatroomListFeedView.kt

```diff
@@ -77,19 +87,70 @@ private fun FeedLoaded(
 ) {
     val listState = rememberLazyListState()
 
-    LazyColumn(
-        contentPadding = PaddingValues(
-            top = 10.dp,
-            bottom = 10.dp
-        ),
-        state = listState
-    ) {
-        itemsIndexed(state.feed.value, key = { index, item -> if (index == 0) index else item.idHex }) { index, item ->
-            ChatroomCompose(
-                item,
-                accountViewModel = accountViewModel,
-                navController = navController
-            )
+    val accountState by accountViewModel.accountLiveData.observeAsState()
+    val account = accountState?.account
+
+    val context = LocalContext.current.applicationContext
+
+    fun markAllAsRead() {
```

Comment:
Not sure what the best placement for this function is, here? Same file but not nested? Extracted to another file?